HW1.py

```diff
@@ -0,0 +1,91 @@
+# функция "transcribe"
+def transcribe(dna):
+    rna = ''
+    for i in dna:
+        if i == 'T':
+            rna += 'U'
+        elif i == 't':
+            rna += 'u'
+        else:
+            rna += i
+    return rna
+
+
+# функция "reverse"
+def reverse(dna):
+    return dna[::-1]
+
+
+# функция "complement"
+def complement(dna):
+    compl_dna = ''
+    compl = dict(A='T', C='G', G='C', T='A', a='t', c='g', g='c', t='a')
+    for i in dna:
+        compl_dna += compl[i]
+    return compl_dna
+
+
+# функция "reverse complement"
+def reverse_complement(dna):
+    compl_dna = ''
+    compl = dict(A='T', C='G', G='C', T='A', a='t', c='g', g='c', t='a')
+    for i in dna:
+        compl_dna += compl[i]
+    return compl_dna[::-1]
+
+
+ok_letters = 'aAcCgGtT'  # буквы, которые встречаются в последовательности ДНК

```

Comment:
Работа с множествами происходить быстрее, если нам нужно проверять на вхождение символов.
```suggestion
ok_letters = set('aAcCgGtT')  # буквы, которые встречаются в последовательности ДНК

```